Source/WebCore/svg/SVGElement.cpp

```diff
@@ -1052,27 +1054,21 @@ AffineTransform SVGElement::localCoordinateSpaceTransform(SVGLocatable::CTMScope
 
 void SVGElement::updateRelativeLengthsInformation(bool hasRelativeLengths, SVGElement& element)
 {
-    // If we're not yet in a document, this function will be called again from insertedIntoAncestor(). Do nothing now.
-    if (!isConnected())
-        return;
-
```

Comment:
This function is called form the overriding function 
```
void updateRelativeLengthsInformation() { updateRelativeLengthsInformation(selfHasRelativeLengths(), *this); }
```
which is called from `svgAttributeChanged()` of many `SVGElement` superclasses like `SVGUseElement::svgAttributeChanged()`.

If the SVGElement is removed from the document and its attribute is changed, like this example:

```
const element = document.getElementById("use-element");
element.remove();
element.setAttribute("x", "10%");
```
So I think removing this if-statement will make us update the relative lengths even though we should not.